ksuid/ksuid.py

```diff
@@ -92,11 +95,36 @@ def datetime(self) -> datetime:
         return datetime.fromtimestamp(unix_time)
 
     @property
-    def timestamp(self) -> int:
-        return int.from_bytes(self.__uid[:TIMESTAMP_LENGTH_IN_BYTES], "big") + EPOCH_STAMP
+    def timestamp(self) -> float:
+        return int.from_bytes(self._uid[: self.TIMESTAMP_LENGTH_IN_BYTES], "big") + EPOCH_STAMP
```

Comment:
also breaks the API but since it was wrong originally I guess this is okay.

Reply:
Yeah, this is why I was OK with this (minor) break.